commit b4a032ef448025b6ae7994a3292b7a6567d5d082
Author: q66 <q66@chimera-linux.org>
Date:   Fri Feb 11 04:59:17 2022 +0100

    disable tests that don't work in sandbox; also multisocketsink is flaky

diff --git a/tests/check/meson.build b/tests/check/meson.build
index f135c1f..f593ab9 100644
--- a/tests/check/meson.build
+++ b/tests/check/meson.build
@@ -85,7 +85,7 @@ if host_machine.system() != 'windows'
 endif
 
 # FIXME: Unstable on Windows
-if build_gstgl and host_machine.system() != 'windows'
+if build_gstgl and host_machine.system() != 'linux'
   base_tests += [
     [ 'libs/gstglapi.c', not build_gstgl, [gstgl_dep]],
     [ 'libs/gstglcolorconvert.c', not build_gstgl, [gstgl_dep, gstglproto_dep]],
@@ -70,8 +70,6 @@ if host_machine.system() != 'windows'
     [ 'elements/libvisual.c', not is_variable('libvisual_dep') or not libvisual_dep.found() ],
     [ 'elements/encodebin.c', not theoraenc_dep.found() or not vorbisenc_dep.found() ],
     [ 'elements/multifdsink.c', not core_conf.has('HAVE_SYS_SOCKET_H') or not core_conf.has('HAVE_UNISTD_H') ],
-    # FIXME: multisocketsink test on windows/msvc
-    [ 'elements/multisocketsink.c', not core_conf.has('HAVE_SYS_SOCKET_H') or not core_conf.has('HAVE_UNISTD_H') ],
     [ 'elements/playbin-complex.c', not ogg_dep.found() ],
     [ 'elements/textoverlay.c', not pango_dep.found() ],
     [ 'elements/theoradec.c', not ogg_dep.found() or not theoradec_dep.found() ],
